### PR TITLE
add placeholder

### PR DESCRIPTION
**Pull request by robhicks**
This adds a conventional placeholder. It looks like I need to run grunt build?

---
Resolve of https://github.com/stevermeister/ngWig/issues/135

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ or via bower:
 
   ```<ng-wig ng-model="text1" source-mode-allowed></ng-wig>```
 
+###Placeholder
```
Comment (stevermeister) [inline README.md]: could you also make plunker with this example please?

**Comment by stevermeister** [comment]
nice addition!

**Comment by robhicks** [comment]
I created a [plunker](https://plnkr.co/edit/yXZkpyHcb47rbXfTHs31?p=preview). And I merged in master. After merging it, I got the tests running. However, the build step is messed up. It includes test describes in dist/ng-wig.js and dist/ng-wig.min.js. I don't know enough about grunt to debug it. Any advice?

File: gruntfile.js [hunk 2]
```diff
@@ -28,9 +28,11 @@ module.exports = function(grunt) {
         files: {
           '<%= distPath %>/ng-wig.js': [
               srcPath + '/javascript/app/ng-wig/ng-wig.js',
-              srcPath + '/javascript/app/ng-wig/*.js',
-              srcPath + '/javascript/app/plugins/formats.ngWig.js',
-              srcPath + '/javascript/app/templates.js']
+              srcPath + '/javascript/app/ng-wig/ng-wig.component.js',
```
Comment (robhicks) [inline gruntfile.js]: explicitly list the files in order.

File: gruntfile.js [hunk 3]
```diff
@@ -131,6 +133,7 @@ module.exports = function(grunt) {
   grunt.registerTask('start', ['html2js', 'watch']);
   grunt.registerTask('install', ['clean:libs', 'copy:dev', 'clean:bower', 'html2js']);
   grunt.registerTask('build', ['html2js', 'copy:dist', 'ngAnnotate', 'babel', 'uglify', 'cssmin', 'bump:patch']);
+  grunt.registerTask('devBuild', ['html2js', 'copy:dist', 'ngAnnotate', 'babel', 'uglify', 'cssmin']);
```
Comment (robhicks) [inline gruntfile.js]: add a devBuild test so the build can be tested without bumping and committing the version.

File: src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js [hunk 4]
```diff
@@ -1,66 +0,0 @@
-describe('component: ngWigPlugin', () => {
```
Comment (robhicks) [inline src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js]: move all tests into subfolders so they aren't picked up in the build.

Comment (bampakoa) [inline src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js]: @robhicks I think it is preferable to have all tests along with the actual code. Please have a look at [Y197](https://github.com/bampakoa/angular-styleguide/tree/master/a1#style-y197) style in Angular 1 StyleGuide.

**Comment by stevermeister** [comment]
hi Rob, sorry I was away for a while, now will take a look what we have in PR.

File: gruntfile.js (hunk 3 above)
Comment (stevermeister) [inline gruntfile.js]: ok

File: src/index.html [hunk 5]
```diff
@@ -30,13 +30,13 @@
 <div class="container">
 
     <h1>One simple WYSIWYG</h1>
-    <ng-wig ng-model="text1" class="editor1" source-mode-allowed></ng-wig>
+    <ng-wig ng-model="text1" class="editor1" source-mode-allowed placeholder="'Enter content here.'"></ng-wig>
```
Comment (stevermeister) [inline src/index.html]: I think you don't need quotes here

File: src/javascript/app/ng-wig/ng-wig.component.js [hunk 6]
```diff
@@ -6,7 +6,8 @@ angular.module('ngWig')
       onPaste: '&',
       buttons: '@',
       beforeExecCommand: '&',
-      afterExecCommand: '&'
+      afterExecCommand: '&' ,
+      placeholder: '<?'
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ng-wig.component.js]: it's good to have value binding here (@)

**Comment by stevermeister** [comment]
@robhicks could I ask you to remove "dist" folder from PR and I think it's ready to be merged

**Comment by stevermeister** [comment]
@robhicks 👍

**Pull request merged**